uint8 type
uint32 waitTime
uint32 range
float64 latitude
float64 longitude
float64 altitude

uint8 TYPE_LAND = 1
uint8 TYPE_TAKEOFF = 2
uint8 TYPE_GOTO = 3
